x-pack/test/api_integration/apis/aiops/index.ts

```diff
@@ -11,7 +11,7 @@ import { FtrProviderContext } from '../../ftr_provider_context';
 
 export default function ({ loadTestFile }: FtrProviderContext) {
   describe('AIOps', function () {
-    this.tags(['ml']);
+    this.tags(['aiops', 'walterra']);
```

Comment:
The `aiops` tag makes sense to me, but `walterra` looks like a left-over debug tag.

Reply:
Thanks for catching that! Fixed in [52c1343](https://github.com/elastic/kibana/pull/135431/commits/52c134308784e818d5808e78c3dbd1c0964e2e0f).